### ISSUE TITLE
[BUG] RAM used was over 3 GB

### ISSUE THREAD

**Issue by key2space** [TheBoredTeam/boring.notch#225]
**Describe the bug**
The app is utilizing crazy amounts of RAM. I took the screenshot below today, but yesterday it was 3 GB which is way to much I think.

**To Reproduce**
I have no clue since I just noticed it in the activity monitor

**Expected behavior**
I remember it wasn't so RAM hungry

**Screenshots or recordings**
<img width="974" alt="Screenshot 2024-11-25 at 13 19 23" src="https://github.com/user-attachments/assets/198677b8-d229-4b38-bac3-b97638976437">
<img width="753" alt="Screenshot 2024-11-25 at 13 20 11" src="https://github.com/user-attachments/assets/e15c8db4-62fe-4296-8b5e-90ff29be424f">

**Comment by key2space** [comment]
<img width="1018" alt="Screenshot 2024-12-04 at 23 55 35" src="https://github.com/user-attachments/assets/640ff7e0-e40d-4f50-8f36-e9787dc8a6f9">

kinda crazy if You ask me

**Comment by key2space** [comment]
Yes indeed, I recommend creating a shortcut for that like this one:
[](https://www.icloud.com/shortcuts/8f58dbbeadcb49919f057e1cc852e7b6)
I just type 'nres' into the spotlight, hit enter and pretend like I don't care

**Comment by RyloRiz** [comment]
Is there an impending fix to this? It's been over a month.

**Comment by theboringhumane** [comment]
> Is there an impending fix to this? It's been over a month.

I'm tending towards and this will be the first thing we resolve.
